packages/dashboard/src/pages/Wallets/components/CollectiblePlaceHolder/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { memo } from 'react'
+import { MiniMaskIcon } from '@dimensiondev/icons'
+import { makeStyles } from '@material-ui/core'
+import { MaskColorVar } from '@dimensiondev/maskbook-theme'
+
+const useStyles = makeStyles(() => ({
+    container: {
+        borderRadius: 8,
+        width: 140,
+        height: 215,
+        backgroundColor: MaskColorVar.lightBackground,
+        display: 'flex',
+        flexDirection: 'column',
+    },
+    placeholder: {
+        width: '100%',
+        height: 186,
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    description: {
+        flex: 1,
+        backgroundColor: MaskColorVar.infoBackground,
+    },
+}))
+
+export const CollectiblePlaceholder = memo(() => {
```

Comment:
Is this component indicates a loading state or an "unknown" state? If the loading state, I'd like to suggest using the `Skeleton` component to make it more living

Reply:
is unknown

Reply:
![image](https://user-images.githubusercontent.com/5390719/121773488-e0ba8400-cbae-11eb-80b8-df469bafa0a5.png)

I think this UI is a bit strange. It looks like the loading state since it doesn't reveal any information about the collectible. Is there any screenshot that what will these components appears in?